plan/do-not-destroy-s3.rego

```diff
@@ -0,0 +1,28 @@
+package spacelift
+
+# This policy is a plan policy, it will validate the resources during the plan phase.
+# More details at: https://docs.spacelift.io/concepts/policy/terraform-plan-policy
+# The "deny" rule fires when a specified resource is being deleted.
+# The result is a formatted message with the address of the offending resource.
+
+deny[sprintf(message, [resource.address])] { 
+  # Define the error message format
+  message := "do not delete %s" 
```

Comment:
The policy name and the description in the README says "destroy" but this message says "delete". We need to be consistent.

Reply:
updated